JS-Algorithms/challenges.js

```diff
@@ -41,7 +49,16 @@ Invoking "circularArray(2)" should return "["Island", "Japan", "Israel", "German
 const COUNTRY_NAMES = ["Germany", "Norway", "Island", "Japan", "Israel"];
 
 const circularArray = (index) => {
-  // YOUR CODE HERE...
+  let positiveIndex = Math.abs(index);
```

Comment:
should positiveIndex value be reassigned? would be better use const instead of let?

Reply:
good way to handle negative indexes though 👍